addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -329,28 +339,37 @@ options.registry.DesignTab = options.Class.extend({
         if (params.cssProperty.includes('color')) {
             value = weUtils.normalizeColor(value);
         }
-        for (const selector of this._getSelectors(params.selectorText)) {
+        const selectors = this._getSelectors(params.selectorText);
+        if (params.cssProperty === 'font-family') {
+            // Ensure font-family gets passed to all descendants.
+            selectors.push(...selectors.map(selector => selector + ' *'));
+        }
+        const firstSelector = selectors[0].replace(CSS_PREFIX, '').trim();
+        for (const selector of selectors) {
+            const important = PRIORITY_STYLES[firstSelector].includes(params.cssProperty) ? ' !important' : '';
```

Comment:
I'm torn. On one hand I'd call it `priority` rather than `important` because sometimes `important` is not `!important`. But on the other hand, it makes it look like there might be more to it while really there is just `!important` or nothing... What do you think ?

Reply:
I'm ok with either honestly. `getPropertyPriority` returns `'important` or `''` so to be coherent with the DOM API I _could_ indeed use `priority` if you prefer.

Reply:
Very good point ! Let's go with `priority` then. 👍